docs/src/index.md

```diff
@@ -431,8 +431,8 @@ The format of the version specifier is described in detail below.
 
 Similar to other package managers, the Julia package manager respects [semantic versioning](https://semver.org/) (semver).
 As an example, a version specifier is given as e.g. `1.2.3` is therefore assumed to be compatible with the versions `[1.2.3 - 2.0.0)` where `)` is a non-inclusive upper bound.
-More specifically, a version specifier is either given as a **caret specifier**, e.g. `~1.2.3`  or a **tilde specifier** `^1.2.3`.
-Caret specifiers are the default and hence `1.2.3 == ~1.2.3`. The difference between a caret and tilde is described in the next section.
```

Comment:
In think the right correction is just to change `1.2.3 == ~1.2.3` to `1.2.3 == ^1.2.3`.